### PR TITLE
feat(FEC-13515): change position and design of the toast comp

### PR DESCRIPTION
**Pull request by lianbenjamin**
### Description of the Changes

- change the default position of a toast to be at the bottom-left side
- add the ability to configure the position of a toast (via the add API)
- align the toast appearance with the design
- make the toast show immediately when it is being added

Solves FEC-13515

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: types/modules.d.ts [hunk 1]
```diff
@@ -445,6 +445,9 @@ declare module "services/floating-manager/floating-manager" {
 declare module "services/toast-manager/models/toast-severity" {
     export type ToastSeverity = 'Info' | 'Success' | 'Warning' | 'Error';
 }
+declare module "services/toast-manager/models/toast-type" {
```
Comment (SivanA-Kaltura) [inline types/modules.d.ts:448]: did you update this file manually or did you run types:generate ?

File: src/services/floating-manager/floating-manager.tsx [hunk 2]
```diff
@@ -201,5 +202,7 @@ export class FloatingManager {
       this._updateCachedCanvas();
       this._updateComponents();
     });
+
+    this._eventManager.listen(kalturaPlayer, ToastEvent.SHOW_TOAST, () => this._updateComponents());
```
Comment (SivanA-Kaltura) [inline src/services/floating-manager/floating-manager.tsx:206]: you can do something like
UiManagerEvent.UPDATE_COMPONENTS
and then its more generic and we'll be able to fire it from other places afterwards without needing to refactor this file

File: src/services/toast-manager/toast-manager.tsx [hunk 3]
```diff
@@ -57,6 +61,7 @@ export class ToastManager {
     this._toasts.push(managedToast);
     this._updateToastsUI();
     this._startDurationTimer(managedToast);
+    this.dispatchToastEvent(ToastEvent.SHOW_TOAST);
```
Comment (SivanA-Kaltura) [inline src/services/toast-manager/toast-manager.tsx:64]: see comment in event handler

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: types/modules.d.ts (hunk 1 above)
Comment (lianbenjamin) [reply types/modules.d.ts:448]: manually

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**